client/core/core.go

```diff
@@ -5262,21 +5310,28 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 		return nil, nil, fmt.Errorf("validateOrderResponse error: %w", err)
 	}
 
+	// TODO: Need xcWallet fields for acceptable SwapConf values: a min
+	// acceptable for security, and even a max confs override to act sooner.
```

Comment:
via `WalletInfo`, I guess?

Reply:
These are the settings that would be wallet/asset-specific, but not internal to to the wallet.  They are used at the level of `Core` (and even higher like the UI).  https://github.com/decred/dcrdex/issues/1900

Reply:
Oh but the default *min* confs would be appropriate to get from the asset code via `WalletInfo` or whatever, yeah.